R/io2.R

```diff
@@ -85,18 +85,18 @@ read_json_duckdb <- function(path, ..., lazy = c(cells = 1e6), options = list())
 #' pass a wildcard or a character vector to the `path` argument,
 #'
 #' @details
-#' By default, a lazy duckplyr frame, with a limit of one million cells, is created.
-#' See the "Eager and lazy" section in [duckdb_tibble()] for details.
+#' By default, a funneled duckplyr frame, with a limit of one million cells, is created.
+#' See the "Funneling" section in [duckdb_tibble()] for details.
 #'
 #' @inheritParams rlang::args_dots_empty
 #'
 #' @param path Path to files, glob patterns `*` and `?` are supported.
 #' @param table_function The name of a table-valued
 #'   DuckDB function such as `"read_parquet"`,
 #'   `"read_csv"`, `"read_csv_auto"` or `"read_json"`.
-#' @param lazy Logical, whether to create a lazy duckplyr frame.
-#'   By default, a lazy duckplyr frame is created.
-#'   See the "Eager and lazy" section in [duckdb_tibble()] for details.
+#' @param funnel Logical, whether to create a funneled duckplyr frame.
+#'   By default, a funneled duckplyr frame is created.
```

Comment:
Mention limitation.

Reply:
what do you mean by limitation here

Reply:
The code you're quoting doesn't have "limitation". Can you please rephrase?

Reply:
This is a note to myself, please ignore.